### DISCUSSION TITLE
Question about Station profile, Station location?

### DISCUSSION THREAD

**Discussion by ei8fdb** [magicbug/Cloudlog#3223] [Q&A] [answered]
I've started to write documentation for the Station Logbook / Station Location parts of Cloudlog. First thing I'm doing is reading through discussions and issues to see what's mentioned about them.

I've come across mentions of Station Profile and, seperately, Station Location.

Current documentation mentions [Station Profiles](https://github.com/magicbug/Cloudlog/wiki/Station-Profiles), but current version mentions Station Locations.

Has the term `Station Profile` been replaced by `Station Location`?

**Comment by magicbug** [comment, marked as answer]
Correct Station Locations replaced it
